blueocean-git-pipeline/src/main/java/io/jenkins/blueocean/blueocean_git_pipeline/GitPipelineCreateRequest.java

```diff
@@ -95,4 +98,45 @@ public BluePipeline create(Reachable parent) throws IOException {
         return null;
     }
 
+    private void validate(String name, BlueScmConfig scmConfig){
```

Comment:
Whats the use case or flow where you need it? with regards to creation flow, it uses credentials API  to  fetch credentials, only valid ones are returned and then its used for rest of the flow.